src/app/pages/MediaArticlePage/PagePromoSections/LastestMediaSection/index.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { PropsWithChildren } from 'react';
+
+import { withKnobs } from '@storybook/addon-knobs';
+import { ToggleContextProvider } from '#app/contexts/ToggleContext';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { MEDIA_ARTICLE_PAGE } from '#app/routes/utils/pageTypes';
+import { ServiceContextProvider } from '../../../../contexts/ServiceContext';
+import { withServicesKnob } from '../../../../legacy/psammead/psammead-storybook-helpers/src';
+import ThemeProvider from '../../../../components/ThemeProvider';
+import { Services } from '../../../../models/types/global';
+
+import LatestMediaSection from '.';
+import hausaArticle from '../../../../../../data/hausa/articles/cxr0765kxlzo.json';
+import tamilArticle from '../../../../../../data/tamil/articles/c84m2jl4dpzo.json';
+import pidginArticle from '../../../../../../data/pidgin/articles/cw0x29n2pvqo.json';
+import { LatestMedia } from './LatestMediaTypes';
+import metadata from './metadata.json';
+
+interface Props {
+  service: Services;
+}
+
+interface ComponentProps extends Props {
+  content: LatestMedia[];
+}
+const Component = ({ service, content }: PropsWithChildren<ComponentProps>) => (
+  <RequestContextProvider
+    isAmp={false}
+    pageType={MEDIA_ARTICLE_PAGE}
+    service={service}
+    pathname=""
+  >
+    <ThemeProvider service={service}>
+      <ServiceContextProvider service={service}>
+        <ToggleContextProvider>
+          <LatestMediaSection content={content} />
+        </ToggleContextProvider>
+      </ServiceContextProvider>
+    </ThemeProvider>
+  </RequestContextProvider>
```

Comment:
Not sure if this is relevant / possible, but all stories now include almost all of the providers - https://github.com/bbc/simorgh/blob/latest/.storybook/preview.js#L557-L570 - which was merged in  #10699 - wonder if you can remove some of the providers from here?

Reply:
Now that I'm looking at the the global provider method, it looks a little complicated then I first thought, there's a bit of overhead with [updating the global values from within a story](https://storybook.js.org/docs/react/essentials/toolbars-and-globals#updating-globals-from-within-an-addon). I'm going to speak with Dario about it, so for now I think we leave it like this if that's okay?

Reply:
That's absolutely fine, I didn't expect you to make the changes as part of this, or only if it was very simple and reduced duplication. Definitely worth knowing for the future if we need to add all the providers to our storybook stories or not (I am also keen to know how this should work going forward!)